restock.lic

```diff
@@ -37,6 +37,7 @@ class Restock
       item['buy_num'] = buy_num
       items_to_restock.push(item)
     end
+    coin_needed += 2502 if UserVars.sun['night'] == true && town == "Shard"
```

Comment:
Could you add a comment to the script explaining why this is necessary?

Reply:
Added.